### DISCUSSION TITLE
This library no longer works on ESP Framework 3.x.x in Arduino IDE 2.3.x

### DISCUSSION THREAD

**Discussion by Knottis** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#650] [Q&A]
This library no longer works on ESP Framework 3.x.x in Arduino IDE 2.3.x
----
### Please don't email me about this issue!..
----

- As you can see, this library is not updated for a long time. 
 
- **_And I also don't think they will update or further develop this library!..._**
 
 
- Those of you who are at summer school to learn Arduino and try out different display types will have to find another library.
 
- Yes, Arduino will automatically update the library and framework , _don't know how this can be turned off_.

----
Does anyone know of a similar library that we can use instead?
---

**Comment by board707** [comment]
Please stop opening new threads on this issue. Your activity already looks like spam
